components/os-installer.sh

```diff
@@ -151,6 +178,8 @@ cp -a /mnt/lib64 /lib64
 cp -a /mnt/sbin/e2fsck /tmp/utils/
 cp -a /mnt/sbin/fsck.ext2 /tmp/utils/
 cp -a /mnt/sbin/fsck.ext4 /tmp/utils/
+cp -a ./curl /mnt/sbin/
+cp -a ./partclone.restore /mnt/sbin/
```

Comment:
hmm, that may be dangerous

Reply:
why do you do that?

Reply:
Will remove. I used this part of code while debugging and testing and forgot to remove.